OutOfSchool/OutOfSchool.AuthCommon/Services/ProviderAdminService.cs

```diff
@@ -237,12 +262,16 @@ async Task<ResponseDto> UpdateProviderAdminOperation(UpdateProviderAdminDto upda
             await using var transaction = await context.Database.BeginTransactionAsync().ConfigureAwait(false);
```

Comment:
(Same code style issue, and I understand that it's not yours, so not insisting on fix) 
`UpdateProviderAdminOperation` takes lines 260-387 and as for me it's just too big. At least it has same `ResponseDto()` fileds setting recurrent pattern.
Again, I'm not insisting on changes right now, but please, don't make such code yourself